service-v2.yaml

```diff
@@ -0,0 +1,15 @@
+name: service-hello
+cluster_env: default_cluster_env_2.4.0_py310:1
```

Comment:
If cluster_env can be defaulted, we should omit it as well and add it under optional fields in the docs

Reply:
i think we default to picking a cluster env that matches ray on your laptop -- this is probably not what we want. I'd rather not default this because it directly effects the success of the example and there is no advantage to defaulting it